README.md

```diff
@@ -64,21 +64,21 @@ If [homebrew]([url](https://brew.sh/)) is installed on your Mac, you can simply
 ```
 brew install qwerty-fr
 ```
-Then to `System Preferences` → `Keyboard`, click `+`, scroll down to `Others` and add `qwerty-fr`. Then restart macOS.
+Then to `System Preferences` → `Keyboard` → `Input Sources`, click `+`, scroll down to `Others` and add `qwerty-fr`. Then restart macOS.
```

Comment:
I also noticed that I didn't get the ability to add the input source before I restarted the Mac, so maybe "restart macOS." should be put before?

Reply:
Interesting I haven't had that issue. Maybe it's a macOS Sonoma issue? Feel free to change it either way!